r/src/arrow_cpp11.h

```diff
@@ -44,9 +44,6 @@
 #define UTF8_MASK (1 << 3)
 #define ASCII_MASK (1 << 6)
```

Comment:
I think you can delete these too, they were only used by the macros you removed

Reply:
Ah right of course